docs/products/mysql/howto/create-tables-without-primary-keys.rst

```diff
@@ -0,0 +1,31 @@
+Create new tables without primary keys
+======================================
+
+Depending on when your Aiven for MySQL service has been created, it may by default not allow creating new tables without primary keys. 
+
+When creating tables without primary keys is not allowed, you will get the following error message:
+
+.. code::
+    Unable to create or change a table without a primary key, when the system variable 'sql_require_primary_key' is set. Add a primary key to the table or unset this variable to avoid this message. Note that tables without a primary key can cause performance problems in row-based replication, so please consult your DBA before changing this setting.
+
+If creating tables without primary keys is prevented and the table that you're trying to create is known to be small you may override this setting and create the table anyway. There are two possible options:
```

Comment:
I would also warn that there are consequences and link back to the replication post where people can understand what is the impact

Reply:
Added, thanks for the suggestion